src/main/java/com/uspray/uspray/service/AuthService.java

```diff
@@ -159,4 +160,16 @@ public void withdrawal(String userId, MemberDeleteDto memberDeleteDto) {
     public DupCheckResponseDto dupCheck(String userId) {
         return new DupCheckResponseDto(memberRepository.existsByUserId(userId));
     }
+
+    public LoginTypeResponseDto loginCheck(String userId) {
+        Member member = memberRepository.getMemberByUserId(userId);
+        if (member.getSocialId() == null) {
+            return LoginTypeResponseDto.builder()
+                .isSocial(false)
+                .build();
+        }
+        return LoginTypeResponseDto.builder()
+            .isSocial(true)
+            .build();
```

Comment:
```suggestion
        return LoginTypeResponseDto.builder()
            .isSocial(member.getSocialId() != null)
            .build();
```
if 문의 결과값이 boolean이니 이런식으로 직접 값을 넣는 것은 어떤가요??

Reply:
네 이 방식이 더 좋은 것 같습니다. 원래 LoginTypeResponseDto에 소셜로그인 타입도 표기하려고 만들어둔건데 지금은 boolean밖에 안들어가서 이렇게 줄이면 될 것 같아요!